app/src/components/inputs/inputFilter/inputFilter.stories.jsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/epam/ReportPortal
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { host } from 'storybook-host';
+import { withReadme } from 'storybook-readme';
+import { EntityItemAttributeValues } from '../../../components/filterEntities';
+import {
+  ENTITY_ATTRIBUTE_VALUES,
+  CONDITION_HAS,
+} from '../../../components/filterEntities/constants';
+import { InputFilter } from './inputFilter';
+import README from './README.md';
+
+const value = 'Adam';
+const filterValues = {};
+const entities = [
+  {
+    id: ENTITY_ATTRIBUTE_VALUES,
+    component: EntityItemAttributeValues,
+    value:
```

Comment:
`{ value: '', condition: CONDITION_HAS }`
These values are static, so no need to check for `filterValues`